cmd/stackit-secrets-manager/cmd/common.go

```diff
@@ -20,3 +22,16 @@ func createClient() (*api.Client, error) {
 	}
 	return apiClient, nil
 }
+
+func parseClaimFromJWT(tokenString string, claim string) (string, error) {
+	parser := jwt.NewParser()
+	token, _, err := parser.ParseUnverified(tokenString, &jwt.MapClaims{})
+	if err != nil {
+		return "", err
+	}
+	claims := *token.Claims.(*jwt.MapClaims)
+	if id, ok := claims[claim]; ok {
+		return id.(string), nil
+	}
+	return "", fmt.Errorf("fClaim \"%s\" not found in the token!", claim)
```

Comment:
We could use %q instead of %s here. %q will automatically print the string quoted, the way we do not need to quote it ourselves. See https://pkg.go.dev/fmt for details. There is also an extra 'f' at the start of the string.